bait-files/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -1,5 +1,7 @@
 package com.google.sps.servlets;
 
+import static com.google.sps.data.Constants.*;
```

Comment:
Import all of the symbols individually instead of using the wildcard.

Reply:
Removed wildcard